web/src/components/feature/FeedFlowInput.tsx

```diff
@@ -0,0 +1,33 @@
+import { Switch, TextField } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+
+const FlexContainer = styled.div`
+  display: flex;
+  gap: 16px;
+`
+
+export const FeedFlowInput = (props: {
+  feedFlow: number
+  feedUnit: string
+  onChangeFeedFlow: (event: React.ChangeEvent<HTMLInputElement>) => void
+  onChangeFeedUnit: (event: React.ChangeEvent<HTMLInputElement>) => void
+}) => {
+  return (
+    <FlexContainer>
+      <TextField
+        id="feed-flow-input"
+        value={props.feedFlow.toString()}
+        label="Feed Flow"
+        unit={props.feedUnit}
+        type="number"
+        onChange={props.onChangeFeedFlow}
```

Comment:
Could also "unpack" the event here. So less has to be done in the parent.
`onChange={(event: ...) => onChangeFeedFlow(Number(event.target.value))}`

Alos, unpacking the "props" is common and nice to do.
`const {onChangeFeddFlow, onChangeFeedUnit} = props`

Reply:
I like that even more!

Reply:
I disagree regarding unpacking the props. personally, I like the fact that all props are grouped together, as it makes it easier to see where a value comes from. Also, you avoid the extra lines of unpacking.

Reply:
Not that important if the component is typed.
But if it's not, unpacking gives a nice overview of which props you have "to work with"

Reply:
True, though in that case I would say the real issue was that the component was missing typing. Given that we always use typing, which we should, unpacking props is unnecessary and messy

Reply:
I think the cleanest structure is something like this
```
type Props = {
  firstProp: string
  secondProp: string
}

const myComponent = (props: Props): void => {
  console.log(props.firstProp)
}

export default myComponent
```
I like the consistency of it. The input type is always `Props`, the inputs are always organized in the `props` object, there are only one component in the file and it is set as default, meaning that you can import like ìmport myComponent from './MyComponent'`

Reply:
The exports are fine. Still, if there are only 2-4 props, I still like them defined in the interface :man_shrugging:

Reply:
Why? Your suggestion allow for inconsistencies.